src/main/java/io/xpire/model/tag/Tag.java

```diff
@@ -57,7 +57,7 @@ public int hashCode() {
      * Format state as text for viewing.
      */
     public String toString() {
-        return '[' + this.tagName + ']';
+        return "[" + this.tagName + ']';
```

Comment:
Darren, is there perhaps a reason why one has double quotes `"["` while the other has single quotes `']'`here?

Reply:
Oops, I do too much Python alr